modules/dnn/src/model.cpp

```diff
@@ -100,8 +106,7 @@ void Model::setInputParams(double scale, const Size& size, const Scalar& mean,
 
 void Model::predict(InputArray frame, OutputArrayOfArrays outs)
 {
-    std::vector<Mat> outputs;
-    outs.getMatVector(outputs);
+    std::vector<Mat> & outputs = *(std::vector<Mat> *)outs.getObj();
```

Comment:
Do we really need this change?

Reply:
Please try this: https://github.com/opencv/opencv/pull/15256/files#diff-6de938636906f03a3c475c556f5c8878R105
and https://github.com/opencv/opencv/pull/15256/files#diff-6de938636906f03a3c475c556f5c8878R26